Tests/youdaTests/ReceiptTest.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  youda
+//
+//  Copyright (c) 2020 Industrial Binaries
+//  MIT license, see LICENSE file for details
+//
+
+import XCTest
+@testable import youda
+
+class ReceiptTest: XCTestCase {
+  private func receiptService() -> ReceiptService {
+    guard
+      let receiptData = Data(base64Encoded: .receipt),
+      let receiptService = try? ReceiptService(deviceID: nil, asn1: receiptData) else {
+      fatalError()
+    }
+    return receiptService
+  }
+
+  func testReceipt() {
+    let receipt = receiptService().receipt
+
+    XCTAssertEqual(receipt.bundleId, .bundleId)
+    XCTAssertEqual(receipt.purchases.count, 13)
+    XCTAssertEqual(receipt.purchases.first?.productIdentifier, .productIdentifier)
+  }
+
+  // TODO: When swift package manager start support resources, uncomment add AppleRootInc certificate
```

Comment:
start`s`

Reply:
not sure what you mean here. Is it `uncomment _and_ add AppleRootInc  certificate`?